doc/tutorials/outsourced-cluster.rst

```diff
@@ -0,0 +1,106 @@
+##################################
+Outsourced Distributed Computation
+##################################
+
+Secure XGBoost is tailored toward an outsourced computation model -- one in which a client with sensitive data wants to outsource computation on the sensitive data to an untrusted server with a trusted hardware enclave. With a lot of training data, a cluster of machines, as opposed to a single server, may have to be used. This tutorial provides an example of such a scenario. 
+
+In this example, the client will encrypt data and send it to each machine in the cluster. The data remains encrypted until it is loaded inside each machine's enclave, at which point it can be decrypted. The cluster will then collectively train a model on the training data following the distributed training pattern outlined in the :doc:`Distributed Secure XGBoost tutorial <distributed>`. Lastly, an encrypted version of the trained model will be saved, available for the client to fetch. 
+
+One node in the cluster will initialize a tracker to start RPC servers on all nodes in the cluster. The tracker is discussed in the :doc:`Distributed Secure XGBoost tutorial <distributed>`.
```

Comment:
I think this might flow better if the previous para(s) introduced the RPC communication between the client and server. E.g. we could end the previous para by saying something like: The client communicates with each machine in the cluster via RPC.

Reply:
Actually, maybe it makes more sense to move this para and the next to the architecture section below...

Reply:
Done